install/helm/agones/templates/crds/_gameserverspecschema.yaml

```diff
@@ -153,6 +153,42 @@ properties:
             type: integer
             title: The initial player capacity of this Game Server
             minimum: 0
+{{- if .featureCountsAndLists }}
+      counts:
```

Comment:
nit: the design doc uses `counter`. Personally I feel it's better since the `counter` object contains the names, capacity, besides count. But not sure whether there were other decisions made on the naming that I wasn't aware of.

Reply:
How about "counters"?

Reply:
counters looks good to me